### PR TITLE
Added Generate Short URL example in Java

### PR DESCRIPTION
**Pull request by Pooja444**
## Github Issue
[Appwrite-4191](https://github.com/appwrite/appwrite/issues/4191)

## Description

This change will add the example of generating a short URL using Java. The supported providers are [tinyurl, bitly]
This change also adds a README for deploying the code using docker.

### Successful execution using bitly

Screen Shot 2022-10-04 at 3 24 00 PM

### Successful execution using tinyurl

Screen Shot 2022-10-04 at 3 23 35 PM

### Empty payload error

Screen Shot 2022-10-04 at 3 30 23 PM

### Empty variables error

Screen Shot 2022-10-04 at 3 32 22 PM

### Empty API key error
Screen Shot 2022-10-04 at 3 33 29 PM

### No provider error

Screen Shot 2022-10-04 at 3 34 24 PM

### No URL error

Screen Shot 2022-10-04 at 3 35 52 PM

### Unsupported provider error

Screen Shot 2022-10-04 at 3 36 59 PM

### Invalid URL error

Screen Shot 2022-10-04 at 3 38 20 PM

### CODE REVIEW

**Comment by Meldiron** [comment]
@Pooja444 , thanks for the PR! 🤯 Please give us some time to review it.

File: java/short_url_generator/README.md
Comment (Meldiron) [inline java/short_url_generator/README.md:5]: Let's please have one more example input with provider `bitly` so others can see what exact name they can use for provider.

File: java/short_url_generator/README.md
Comment (Meldiron) [inline java/short_url_generator/README.md:16]: Let's please have one more output example where`success`  is `false`, and there is a `message`.

File: java/short_url_generator/README.md
Comment (Meldiron) [inline java/short_url_generator/README.md:30]: Let's please separate this into 2 API keys - one for Bitly, one for TinyURL. That will allow user use both at the same time.

File: java/short_url_generator/Index.java
Comment (Meldiron) [inline java/short_url_generator/Index.java:195]: This needs to change a little due to requested change in README. We should have 2 separate variables for each API key - for each provider. Then we only need to validate API key of a specific provider - that would either be `TINYURL_API_KEY` or `BITLY_API_KEY`.

File: java/short_url_generator/README.md
Comment (Meldiron) [reply java/short_url_generator/README.md:30]: We could call them `TINYURL_API_KEY` and  `BITLY_API_KEY`.

File: java/short_url_generator/Index.java
Comment (Meldiron) [inline java/short_url_generator/Index.java:62]: This will need to be changed to either get `TINYURL_API_KEY` or `BITLY_API_KEY`.

**Review by Meldiron — CHANGES_REQUESTED** [review]
Great quality of Java code! 🤯 I left my review, a few small things need to be changed, but other than that, amazing PR. Once my PRs are addressed, I can mark this approved.

**Comment by Pooja444** [comment]
@Meldiron thanks a lot for your comments! I have addressed them and made the required changes.

**Comment by Pooja444** [comment]
Tested the changes too -

TinyUrl successful execution - 

Screen Shot 2022-10-07 at 10 20 44 AM


Bitly successful execution - 

Screen Shot 2022-10-07 at 10 20 04 AM


TinyUrl key not passed error -

Screen Shot 2022-10-07 at 10 21 02 AM


Bitly key not passed error -

Screen Shot 2022-10-07 at 10 22 10 AM

**Review by Meldiron — APPROVED** [review]
Looks great to me 😇 🎉🎉🎉

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**
